composer/datasets/mnist.py

```diff
@@ -16,7 +16,7 @@
 class MNISTDatasetHparams(DatasetHparams, SyntheticHparamsMixin):
     """Defines an instance of the MNIST dataset for image classification.
 
-    Parameters:
+    Args:
```

Comment:
References to `DataSpec` in the "Returns" section for the `initialize_object` method aren't resolving.